util/util_test.go

```diff
@@ -36,3 +36,15 @@ func (s *TestSuite) TestNamespaceExecutor(c *C) {
 	_, err = ne.Execute("mount", []string{})
 	c.Assert(err, IsNil)
 }
+
+// TestFindDockerdProcess can pass in dapper
+// ENV DAPPER_RUN_ARGS --privileged -v /dev:/host/dev -v /proc:/host/proc
```

Comment:
Don't need the comment. `go-iscsi-helper` way of building is by `make`, not `go build && go test`. As long as `make` works, it's fine.